samples/ace-teams-covid/covid-spfx/src/webparts/covidAdmin/components/atoms/Button.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { Logger, LogLevel } from "@pnp/logging";
+import { isEqual } from "@microsoft/sp-lodash-subset";
+
+export interface IButtonProps {
+  className: string;
+  disabled: boolean;
+  label: string;
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+export interface IButtonState {
+}
+
+export class ButtonState implements IButtonState {
```

Comment:
It could but I don't see the point in removing them, and I think it just confuses people who aren't extremely React savvy. Unless you have some unknown to me performance reason?  Also, in the end this is going to be updated to inherit from the HTWOO React library, so they will go away.

Reply:
Fair enough - I doubt there's a performance difference too.